### PR TITLE
Add circleci build

### PR DESCRIPTION
**Pull request by guillaumerose**
In Docker for Mac, we package qcow-tool ourself. It will be better to compile it in a proper build. 
This is the first step for getting this artifact. 

For the libev dependency, I don't really know what should I do. I tried a lot of things to get the dylib compiled with the correct flags without luck. Can I get help on this ? Thanks!

### CODE REVIEW

**Comment by samoht** [comment]
@djs55 do we really need libev for `qcow-tool`? I guess using the standard scheduler should work as well and won't require a depext.

@guillaumerose so the fix is to just remove mentions of libev completely...

**Comment by djs55** [comment]
Yes good idea @samoht. I can take a look at this later if you like.

**Comment by guillaumerose** [comment]
Thanks I will do that.

**Comment by guillaumerose** [comment]
I updated the file. It's better now.

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  environment:
+    MACOSX_DEPLOYMENT_TARGET: "10.10"
+    OPAMYES: "1"
+compile:
+  override:
+    - mkdir build/
+    - brew install opam
+    - opam init
+    - opam pin add qcow .
+    - eval `opam config env` && cp `which qcow-tool` build/
```
Comment (samoht) [inline circle.yml]: if you want to avoid subshells you can also write: `opam config exec -- cp `which qcow-tool` build/`

**Comment by djs55** [comment]
LGTM!

**Pull request merged**